ISSUE-TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+**Category:** 
```

Comment:
I'm thinking maybe it would be good to provide some options for this one?

Reply:
Indeed it would be
I suggest **Category: (API, Account, Web, App)**